SeriousGame/Assets/Custom/Scripts/GameController.cs

```diff
@@ -50,17 +57,19 @@ private void OnDisable()
         {
             EnergyLevel = EnergyLevel,
             FoodLevel = FoodLevel,
-            WaterLevel = WaterLevel
-        };
+            WaterLevel = WaterLevel,
+            OldDate = System.DateTime.Now
+
+    };
```

Comment:
Indention gone wrong, please run: `Ctrl + K + D` (assuming u use VS 2017) to format the code.

Reply:
will incorporate that in the next commit which will include content as well

Reply:
Okay, will wait for the next commit. Please also do this for AdjustDataScript.cs as well. 👍